weather_mv/loader_pipeline/sinks.py

```diff
@@ -338,15 +336,27 @@ def __open_dataset_file(filename: str,
         False)
 
 
+def copy(src: str, dst: str) -> None:
+    """Copy data via `gcloud alpha storage` or `gsutil`."""
+    errors = []
+    for cmd in ['gcloud alpha storage cp', 'gsutil cp']:
+        try:
+            subprocess.run(cmd.split() + [src, dst], check=True, capture_output=True)
+            return
+        except subprocess.CalledProcessError as e:
+            errors.append(e)
+
+    msg = f'Failed to copy file {src!r} to {dst!r}'
+    logger.error(f'{msg} due to {errors[0].stderr.decode("utf-8")}')
```

Comment:
Maybe we should log `errors[-1]` instead?

Reply:
Actually, if both commands are failing, we probably want to log that we've tried both commands and also log both error messages?